testing/integration_test.go

```diff
@@ -74,9 +74,10 @@ retry curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.
 export NVM_DIR="$HOME/.nvm" >/dev/null
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" >/dev/null
 
-# nvm install writes to stderr and stdout on successful install, so both are
+# nvm install writes to stderr and stdout on successful install, so some
+# output will appear on successful installs. Only output to stdout is
```

Comment:
"Only output to stdout is redirected" - this documents "what", not "why". Please add a note on "why".

Reply:
Done.